ckb-bin/src/subcommand/migrate.rs

```diff
@@ -24,16 +24,10 @@ pub fn migrate(args: MigrateArgs) -> Result<(), ExitCode> {
                 return Err(ExitCode::Failure);
             }
 
-            if args.check {
-                if matches!(db_status, Ordering::Less) {
-                    return Ok(());
-                } else {
-                    return Err(ExitCode::Cli);
-                }
-            }
-
             if matches!(db_status, Ordering::Equal) {
                 return Ok(());
+            } else if args.check {
```

Comment:
The `ckb migration --check`'s description is "Perform database version check without migrating, if migration is in need ExitCode(0) is returned，otherwise ExitCode(64) is returned". 
So, when database version is equal, `ExitCode(64)` is expected.

Reply:
It's for automation. For example:

```
if ckb migrate --check; then
  ckb migrate --force
fi
```

Reply:
yes, got it in the document: 
```console
Add command `ckb migrate --check`. If migration is in need 0 will be return，otherwise 64.
```
thanks.